cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,163 @@
+| 1               | `alert`        | 1             |
+| 2               | `critical`     | 2             |
+| 3               | `error`        | 3             |
+| 4               | `warning`      | 4             |
+| 5               | `notice`       | 5             |
+| 6               | `info`         | 6             |
+| 7               | `debug`        | 7             |
+
+#### Windows Event Log
+Binding SHOULD NOT depend on the internal `Level` integer value of the log 
+record as it is defined by the windows event producer. 
+
+Instead, cloudevent producers SHOULD use the following `loglevelnum`s instead of 
+the windows event log record `Level` values.
+
+| Windows Event Level | `loglevelname` | `loglevelnum` |
+| ------------------- | -------------- | ------------- |
+| `Critical`          | `critical`     | 2             |
+| `Error`             | `error`        | 3             |
+| `Warning`           | `warning`      | 4             |
+| `Information`       | `info`         | 6             |
+| `Verbose`           | `verbose`      | 8             |
+
+#### Python
+| Python Log Level | `loglevelname` | `loglevelnum` |
+| ---------------- | -------------- | ------------- |
+| `CRITICAL`       | `critical`     | 2             |
+| `FATAL`          | `critical`     | 2             |
+| `ERROR`          | `error`        | 3             |
+| `WARN`           | `warning`      | 4             |
+| `WARNING`        | `warning`      | 4             |
+| `INFO`           | `info`         | 6             |
+| `DEBUG`          | `debug`        | 7             |
+| `NOTSET`         | `verbose`      | 8             |
+
+#### Java (Spring)
+| Sprint Log Level | `loglevelname` | `loglevelnum` |
+| ---------------- | -------------- | ------------- |
+| `ERROR`          | `error`        | 3             |
+| `WARN`           | `warning`      | 4             |
+| `INFO`           | `info`         | 6             |
+| `DEBUG`          | `debug`        | 7             |
+| `TRACE`          | `verbose`      | 8             |
+
+#### .NET (Serilog)
+| Serilog Log Level | `loglevelname` | `loglevelnum` |
+| ----------------- | -------------- | ------------- |
+| `Fatal`           | `critical`     | 2             |
+| `Error`           | `error`        | 3             |
+| `Warning`         | `warning`      | 4             |
+| `Information`     | `info`         | 6             |
+| `Debug`           | `debug`        | 7             |
+| `Verbose`         | `verbose`      | 8             |
+
+#### Javascript (Winston)
+| Winston Log Level | `loglevelname` | `loglevelnum` |
+| ----------------- | -------------- | ------------- |
+| `error`           | `error`        | 3             |
+| `warn`            | `warning`      | 4             |
+| `info`            | `info`         | 6             |
+| `http`            | `http`         | 7             |
+| `verbose`         | `verbose`      | 8             |
+| `debug`           | `debug`        | 9             |
```

Comment:
is it ok that debug is 7 in the other ones?

Reply:
yes